Update repository links from vivax3794/natrix to Serpent-Tools/natrix

Shreyanshsingh23 opened the pull request:
# Update repository links from vivax3794/natrix to Serpent-Tools/natrix

## Description

This PR updates all repository links throughout the codebase to reflect the new organization structure, changing references from `vivax3794/natrix` to `Serpent-Tools/natrix`.

## Changes Made

### Files Updated (5 files total):

1. **`crates/natrix/Cargo.toml`**
   - Updated `repository` field from `https://github.com/vivax3794/natrix` to `https://github.com/Serpent-Tools/natrix`

2. **`crates/natrix_macros/Cargo.toml`**
   - Updated `repository` field from `https://github.com/vivax3794/natrix` to `https://github.com/Serpent-Tools/natrix`

3. **`crates/natrix_macros/src/lib.rs`**
   - Updated documentation link in crate-level comment from `https://github.com/vivax3794/natrix` to `https://github.com/Serpent-Tools/natrix`

4. **`crates/natrix/src/dom/html_elements.rs`**
   - Updated documentation URL from `https://vivax3794.github.io/natrix/reactivity.html` to `https://serpent-tools.github.io/natrix/reactivity.html`

5. **`docs/src/README.md`**
   - Updated logo image URL from `https://github.com/vivax3794/natrix/raw/master/assets/logo.png` to `https://github.com/Serpent-Tools/natrix/raw/master/assets/logo.png`

## Impact

- **Repository consistency**: All links now point to the correct organization
- **Documentation accuracy**: Users will be directed to the correct repository and documentation
- **Package metadata**: Cargo.toml files now have accurate repository information
- **No functional changes**: This is purely a URL update with no impact on code functionality

## Verification

- ✅ Searched entire codebase for any remaining `vivax3794/natrix` references - none found
- ✅ Verified all new `Serpent-Tools/natrix` links are correctly in place
- ✅ No linting errors introduced
- ✅ All changes are minimal and focused on URL updates only

## Related

Fixes #92

## Testing

No testing required as this only updates URLs and metadata. The changes are:
- Repository links in Cargo.toml files
- Documentation links in source code comments
- Image URLs in markdown files

All changes maintain the same functionality while pointing to the correct repository location.

Shreyanshsingh23 committed:
Update repository links from vivax3794/natrix to Serpent-Tools/natrix

- Updated Cargo.toml repository fields in natrix and natrix_macros crates
- Updated documentation links in README.md and lib.rs files
- Updated documentation URL in html_elements.rs from vivax3794.github.io to serpent-tools.github.io

Fixes #92

vivax3794 commented on docs/src/README.md:
```diff
@@ -1,6 +1,6 @@
 # Introduction 
 
-<img src="https://github.com/vivax3794/natrix/raw/master/assets/logo.png" alt="Logo" width="300" height="300">
+<img src="https://github.com/Serpent-Tools/natrix/raw/master/assets/logo.png" alt="Logo" width="300" height="300">
```
This part should match the repo readme, forgot to update it when the logo position changed, prolly best to just do it in this pr.
```suggestion
<img src="https://raw.githubusercontent.com/Serpent-Tools/branding/refs/heads/main/natrix.svg" alt="Logo" width="300" height="300">
```

Shreyanshsingh23 replied:
No worries @vivax3794 , I will do it in this PR only.

Shreyanshsingh23 committed:
Update logo URL to use branding repository

- Changed from natrix/assets/logo.png to branding/natrix.svg
- Now uses Serpent-Tools/branding repository
- Matches the main repository README logo location

vivax3794 reviewed (approved):
Nice, Thanks!
